src/Symfony/Component/Intl/Tests/LanguagesTest.php

```diff
@@ -943,6 +1535,8 @@ public function testAlpha3CodeExists()
     {
         $this->assertTrue(Languages::alpha3CodeExists('nob'));
         $this->assertTrue(Languages::alpha3CodeExists('nld'));
+        $this->assertTrue(Languages::alpha3CodeExists('ace'));
```

Comment:
Please also test that the following (problematic) alpha3 codes exist: `nor`, `twi` and `tgl`.

Mentioned in #33136

Reply:
right! thanks. In fact we miss those alpha3 codes currently, due `nob => nb` as well a `nob => nb`. Ive got a patch locally.. 

`tgl` is an interesting case, it seems to be an alias actually (so alpha3 to alpha3): https://github.com/unicode-org/icu/blob/d1293c50cbef31156f3d28f16cc67264389fd8fc/icu4c/source/data/misc/metadata.txt#L1374-L1377

not sure what to do with, as we just cleaned up "aliases" (#33139), a concept that never worked. But im not sure we should introduce it :/

Reply:
fixed